.taurignore

```diff
@@ -0,0 +1,10 @@
+packages/api
+packages/cli
+crates/tauri-cli
+crates/tauri-bundler
+crates/tauri-driver
+crates/tauri-macos-sign
+crates/tauri-schema-generator
+crates/tests
+bench
+.changes
```

Comment:
if we ignoring .changes, should we also ignore `.github`, `.docker` and other directories ?

Reply:
how often do we change these anyway? 😅

Reply:
never but this is about local development and it could either be accidentally saved or you could be changing a workflow while working on a feature, I'd say add them just in case

Reply:
cool will do that tomorrow when i wake up